### PR TITLE
Disables "Go to definition" button when hovering at definition

### PR DESCRIPTION
**Pull request by Numbers88s**
Disables "Go to definition" button when already at definition

### CODE REVIEW

File: client/shared/src/api/protocol/contribution.ts [hunk 1]
```diff
@@ -84,6 +84,9 @@ export interface ActionContribution {
     /** The title that succinctly describes what this action does. */
     title?: TemplateExpression
 
+    /** The title that succinctly describes what it is when the action is disabled. */
```
Comment (efritz) [inline client/shared/src/api/protocol/contribution.ts:87]: This title describe what _what_ is?

File: client/shared/src/hover/actions.ts [hunk 2]
```diff
@@ -445,6 +462,7 @@ export function registerHoverContributions({
                             action: 'findReferences',
                             when:
                                 'findReferences.url && (goToDefinition.showLoading || goToDefinition.url || goToDefinition.error)',
+                            disabledWhen: 'false',
```
Comment (chrismwendt) [inline client/shared/src/hover/actions.ts:465]: You can drop this, right? I imagine it defaults to `undefined`, which is falsy.

Comment (Numbers88s) [reply client/shared/src/hover/actions.ts:465]: yeah definitely. I like being more explicit though. Would you like it to be undefined/implicit?

Comment (chrismwendt) [reply client/shared/src/hover/actions.ts:465]: Nah, no need to change it

**Review by chrismwendt — APPROVED** [review]
🚀

File: client/shared/src/api/protocol/contribution.ts (hunk 1 above)
Comment (Numbers88s) [reply client/shared/src/api/protocol/contribution.ts]: updated! ✅

**Pull request merged**